Examples/Physics_applications/capacitive_discharge/PICMI_inputs_1d.py

```diff
@@ -9,10 +9,132 @@
 
 import numpy as np
 from pywarpx import callbacks, fields, picmi
+from scipy.sparse import csc_matrix
+from scipy.sparse import linalg as sla
 
 constants = picmi.constants
 
 
+class PoissonSolver1D(picmi.ElectrostaticSolver):
```

Comment:
This can be removed and the MLMG solver used once https://github.com/ECP-WarpX/WarpX/issues/3123 is addressed.

Reply:
:+1: Let's also note this inline:
```suggestion
class PoissonSolver1D(picmi.ElectrostaticSolver):
    """This can be removed and the MLMG solver used once
       https://github.com/ECP-WarpX/WarpX/issues/3123 is addressed."""
```